app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1506,6 +1506,22 @@ def _validate_rollout_version_support(self, data):
 
         return data
 
+    def _validate_bucket_duplicates(self, data):
+        if not self.instance or not self.instance.is_rollout:
+            return data
+        is_rollout = self.instance.is_rollout
+        count = NimbusExperiment.objects.filter(
+            channel=self.instance.channel,
```

Comment:
❗ good point. Do you think we also need to worry about things that are in review that haven't been launched yet?

Reply:
Hrmmm let's think.  

If there's one live, and one in draft, it'll get the warning.

If there's two in draft, should they both warn about each other?

I think no..... since things can live in draft indefinitely then you'd just always get the warning.  If two are in draft, no warning, one goes live, now the unlaunched one gets the warning.  I think that's sufficient.  What do you think?

Reply:
nit pick: you can save self.instance.is_rollout in a variable and then can use like this 
```python   
is_rollout =  self.instance.is_rollout    
if not self.instance or not is_rollout:
            return data